services/application.js

```diff
@@ -318,8 +334,12 @@ class Application {
 
     async reopenApplication(document, context) {
         verifySession(context)
-            .verifyInitialized()
-            .verifyPermission(USER_PERMISSION_CONSTANTS.SUBMISSION_REQUEST.CREATE);
+            .verifyInitialized();
+        const userScope = await this.#getUserScope(context?.userInfo, USER_PERMISSION_CONSTANTS.SUBMISSION_REQUEST.CREATE);
```

Comment:
This should be just check if user is creator of the application, not create permission.

Reply:
Yes, updated the submission request creator only.